BankClient/Main.cs

```diff
@@ -0,0 +1,25 @@
+﻿// See https://aka.ms/new-console-template for more information
```

Comment:
You should rename this file back to Program.cs

Reply:
I renamed